packages/client/modules/userDashboard/components/UserTasksHeader/UserTasksHeader.tsx

```diff
@@ -50,6 +50,8 @@ const StyledCheckbox = styled(Checkbox)({
 
 const UserTasksHeaderDashSectionControls = styled(DashSectionControls)({
   justifyContent: 'flex-start',
+  flexWrap: 'wrap',
+  flexShrink: 0,
```

Comment:
`flexShrink` goes on the individual components that you don’t want to shrink, not their container. I moved this in a separate commit.